_data/companies.csv

```diff
@@ -14,14 +16,20 @@ NamingMatters,http://www.namingmatters.com/,SaaS products,"Trademark pre-clearan
 PlaceOS,https://place.technology/,SaaS products,"Connects everything in your office, tenancy or building to create a seamless user experience"
 Rainforest QA,https://www.rainforestqa.com/,SaaS products,"QA at the Speed of Development, Intelligent Crowdtesting for Agile Teams"
 Void,https://voidapp.co/,SaaS products,"Read + Retain in a distracted world"
+Aveine,https://www.aveine.com/,Software Development,"The first smart wine aerator"
 Bulutfon,https://www.bulutfon.com/,Software Development,"Cloud Voice & VOIP solution for Turkey"
 Duo Design,https://www.duodesign.co.uk/,Software Development,"Web and application design and development from Manchester"
+Kemal,https://kemalcr.com/,Software Development,"Fast, Effective, Simple Web Framework"
+Kinko booking,https://kinkobooking.com,Software Development,"Gérez toutes vos Réservations en un Seul Click"
```

Comment:
The description is not much use for non-French speakers and it's a non-descriptive marketing slogan.
There's a much more meaningful definition on their homepage:
> Kinko est votre nouvelle plateforme de réservation en Real Time entre Hôtels / Maisons d'hôtes et Agences de voyages. 
Which I'd loosely translate as this:

```suggestion
Kinko booking,https://kinkobooking.com,Software Development,"A platform for managing real-time reservations in hotels and travel agencies."
```